steely/plugins/deepdream.py

```diff
@@ -0,0 +1,51 @@
+'''
+http://i0.kym-cdn.com/photos/images/original/001/042/049/5ef.jpg
+'''
+import requests
+from steely import config
+
+__author__ = 'CianLR'
+COMMAND = '..'
+API_URL = "https://api.deepai.org/api/deepdream"
+
+def get_dream(url):
+    return requests.post(
+        API_URL,
+        data = {'content': url},
+        headers = {'api-key': config.DEEP_API_KEY},
+    ).json()
+
+def grab_image(message):
+    for a in message.attachments:
+        # TODO: Something less stupid than below
+        if a.__class__.__name__ == 'ImageAttachment':
```

Comment:
wtf

Reply:
trying something like `isinstance(a, fbchat.models.ImageAttachment)` but can't import it